gateways/istio-egress/templates/_helpers.tpl

```diff
@@ -3,7 +3,8 @@
 Expand the name of the chart.
 */}}
 {{- define "gateway.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{ $gateway := index .Values "gateways" "istio-egressgateway" }}
```

Comment:
Please get rid of this - no need to use a tempalate to get the name, we no longer have gateways.